libobs/util/cf-lexer.c

```diff
@@ -71,22 +71,26 @@ char *cf_literal_to_str(const char *literal, size_t count)
 
 	if (count < 2)
 		return NULL;
-	if (literal[0] != literal[count-1])
+	if (literal[0] != literal[count - 1])
```

Comment:
Although your change here is how it should be, this code styling change is unnecessary and should be removed from this commit.